tests/e2e/cucumber/steps/ui/links.ts

```diff
@@ -21,8 +21,13 @@ When(
 )
 
 When(
-  '{string} creates a public link for the space with password {string} using the sidebar panel',
-  async function (this: World, stepUser: string, password: string): Promise<void> {
+  '{string} creates a public link for the space {string} with password {string} using the sidebar panel',
```

Comment:
let's not change this step.
```suggestion
  '{string} creates a public link for the space with password {string} using the sidebar panel',
```